tests/unit/strings/text-test.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+namespace Jsys\strings\Test;
+
+use jsys\types\strings\Text;
+use PHPUnit\Framework\TestCase;
+
+class TextTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp(); // TODO: Change the autogenerated stub
+    }
+
+    public function test_must_accept_the_empty_string()
```

Comment:
`test_must_accept_the_empty_string`

we are not talking about validation or acceptance here